docs/composedb/guides/composedb-server/running-in-the-cloud.mdx

```diff
@@ -9,7 +9,15 @@ Run a ComposeDB server in the cloud
 
 :::danger
 
-To run a Ceramic node in production, it is critical to persist the [Ceramic state store](../../../protocol/js-ceramic/guides/ceramic-nodes/running-cloud#ceramic-state-store), [IPFS datastore](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#datastorespec) as well as Posgres database. The form of storage you choose should also be configured for an emergency recovery with data redundancy, and some form of snapshotting and/or backups. **Loss of this data can result in permanent loss of Ceramic streams and will cause your node to be in a corrupt state.**
+To run a Ceramic node in production, it is critical to persist the [Ceramic state store](../../../protocol/js-ceramic/guides/ceramic-nodes/running-cloud#ceramic-state-store), [IPFS datastore](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#datastorespec) as well as Postgres database. The form of storage you choose should also be configured for an emergency recovery with data redundancy, and some form of snapshotting and/or backups. **Loss of this data can result in permanent loss of Ceramic streams and will cause your node to be in a corrupt state.**
+
+Your backup cadence should implement the following order:
```

Comment:
huh, I was going to say that this isn't using the word "cadence" right, but I googled it and it does indeed seem that there is a definition related to word ordering!   Thanks for teaching me something new today!  I do still think of "cadence" as being more often about the rhythm/tempo than the ordering though.

In any case, I think just changing the word to "process" will make the sentence clearer.